jo-rdan/Free-Mentor
# #167603588 build signup page

#### What does this PR do?
It creates a user interface signup page.

#### Description of Task to be completed
- It creates a signup form for user interface.
- It creates a responsive signup page.

#### How should this be manually tested?
Clone this repo on your local machine.
If you have a modern editor such as vscode, download the live server extension
- Choose **Open with live server** and it should open in your browser
If you don't have modern editors, you can open the signup file locally on your computer.

#### Any background context you want to provide?
This is just a user interface, there is no core functionality implemented yet

#### What are the relevant pivotal tracker stories?
[#167603588](https://www.pivotaltracker.com/story/show/167603588)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/51836855/62423689-92fe9780-b678-11e9-9c9f-7d8bafdb3ae5.png)
